src/main/proto/com/google/api/codegen/config.proto

```diff
@@ -168,19 +170,27 @@ message CollectionConfigProto {
   // empty, no name methods are generated.
   string name_pattern = 1;
 
-  // Name to be used as a basis for generated methods.
+  // Name to be used as a basis for generated methods and classes.
   string entity_name = 2;
 }
 
 message CollectionOneofProto {
-  // Name to be used as a basis for generated methods.
+  // Name to be used as a basis for generated methods and classes.
   string oneof_name = 1;
 
   // A list of the entity names of the CollectionConfigs to be included in the
   // oneof collection.
   repeated string collection_names = 2;
 }
 
+message FixedCollectionProto {
```

Comment:
This isn't really a "collection". Maybe call this `FixedResourceNameValueProto`. I know you're going to fix the field names and proto names containing "collection" later, but it is an especially bad fit for this case, so we might as well start this one right.